src/types.js

```diff
@@ -380,21 +558,46 @@ class FP_Time extends FP_TimeBase {
   }
 
 
+
+
+
+
   /**
    *  Returns a new Date object for a time equal to what this time would be if
```

Comment:
Would help to document the fact that the "date" part of the returned date object is randomly chosen and is therefore insignificant, if that's the case.

Reply:
Agreed.  Done.